cmdstanpy/utils/filesystem.py

```diff
@@ -169,11 +170,15 @@ def __exit__(self, exc_type, exc_val, exc_tb) -> None:  # type: ignore
 
 
 class SanitizedOrTmpFilePath:
-    """Context manager for tmpfiles, handles spaces in filepath."""
+    """
+    Context manager for tmpfiles, handles special characters in filepath.
+    """
+    PATTERN = re.compile(r"[\s~]")
```

Comment:
This doesn't need to be a member variable, could just be outside the class and avoid the `self.`s later

Reply:
Makes sense. My motivation for the static variable was to keep everything in one place. Happy to change it though.